traveler-ios-ui/TravelerKitUI/ViewControllers/BookableAvailabilityViewController.swift

```diff
@@ -10,7 +10,7 @@ import UIKit
 import TravelerKit
 
 protocol BookableAvailabilityViewControllerDelegate: class {
-    func bookableAvailabilityViewControllerDidConfirm(_ controller: BookableAvailabilityViewController, bookingForm: BookingForm)
+    func bookableAvailabilityViewControllerDidReceiveCheckout(withForm form: BookingForm?)
```

Comment:
You might want to change the naming of your parameters. Instead of (withForm form: BookingForm?) you can do the following (with form: BookingForm). This way when the method is called it can be read like english i.e. --> (with: BookingForm). 

Also, why is the BookingForm optional?. if the checkout was received then the booking form should be there, right?.

Reply:
Also, pass the view controller the same way it is being passed in the line that you deleted. This is because we want our VC's to be reusable and it should be the responsibility of the VC conforming to the delegate to know what to do (if anything) with the delegating VC.